samples/snippets/requirements.txt

```diff
@@ -1,2 +1,6 @@
+# Version 2.1.1 of google-api-core version is the first type-checked release.
+google-api-core[grpc]==2.1.1
```

Comment:
Shouldn't this be a minimum pin?

Reply:
Hmm... I think this should actually be moved into the `mypy_samples` nox session, as the change is primarily meant for the type check's benefits.

Interestingly though, the samples tests already **don't** use the minimum version - they just [install stuff](https://github.com/googleapis/python-pubsub/blob/17b5d143f2ffcab7e7efea8bb2018148c00ce1ee/samples/snippets/noxfile.py#L191-L195) from `requirements.txt` (with no constraints on api-core), and by default `pip` installs the most recent compatible versions.

I'll check with the samples team on that.

Reply:
Got a response, the samples are actually tested with the latest library release. Sounds like the version pin should just be updated to `>=` (or even be omitted altogether, but I prefer to be explicit here).